Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -151,56 +152,58 @@ window.__firefox__.execute(function($) {
     return false
   }
   
+  const makeStyleSheet = () => {
+    const style = document.createElement('style')
+    style.setAttribute('type', 'text/css')
+    return style
+  }
+  
   /// Create the style sheet if it isn't already created
   /// We do this in iOS here because we can't initialize a style sheet
   const ensureStyleSheet = () => {
     if (CC.cosmeticStyleSheet === undefined) {
-      const style = document.createElement('style')
-      style.setAttribute('type', 'text/css')
-      document.head.appendChild(style)
-      CC.cosmeticStyleSheet = style.sheet
+      const styleElm = makeStyleSheet()
+      document.body.appendChild(styleElm)
+      CC.cosmeticStyleSheet = styleElm
     }
   }
   
   /// Takes selectors and adds them to the style sheet
-  const processSelectors = (selectors) => {
-    ensureStyleSheet()
-    let nextIndex = CC.cosmeticStyleSheet.rules.length
-
+  const processHideSelectors = (selectors) => {
     selectors.forEach(selector => {
-      if ((typeof selector === 'string') && (CC.hide1pContent || !CC.allSelectorsToRules.has(selector))) {
+      if ((typeof selector === 'string') && (CC.hide1pContent || !CC.allHideSelectorsToRules.has(selector))) {
         const rule = selector + '{display:none !important;}'
-        CC.cosmeticStyleSheet.insertRule(rule, nextIndex)
+        CC.allRules.push(rule)
         
         if (!CC.hide1pContent) {
-          CC.allSelectorsToRules.set(selector, nextIndex)
+          CC.allHideSelectorsToRules.set(selector, CC.allRules.count - 1)
```

Comment:
`CC.allRules` is an array according to line `51`. There is no such thing as `Array.count` in Javascript.

Reply:
Ahh good catch. This script is yet acting on this part of the function because it doesn't support non-aggressive mode yet so it's not a problem however it should be fixed.